packages/ios-client/project.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
is this file needed?

Reply:
This package is created by script `gen-mocks.sh`. It is necessary for iOS tests. Not sure though is necessary in repo. On the other hand it affects testing soo i think it should be in repo. If you think the opposite give me your thoughts.

Reply:
Never mind. Deleted.